rascal-vscode-extension/.vscode/launch.json

```diff
@@ -16,9 +16,24 @@
 			"outFiles": [
 				"${workspaceFolder}/out/**/*.js"
 			],
-            "env": {
-                "RASCAL_LSP_DEV": "true"
-            },
+				"env": {
+					"RASCAL_LSP_DEV": "true"
+				},
```

Comment:
formatting went haywire

Reply:
Ah, this was already a mess with mixed tabs and spaces, I thought I fixed it.